integrations/crowdnode/src/main/res/layout/dialog_staking.xml

```diff
@@ -14,55 +14,248 @@
+
+                <ImageView
+                    android:id="@+id/staking_leaving_the_pool"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="15dp"
+                    android:src="@drawable/ic_steaking_leaving_pool"
+                    app:layout_constraintStart_toStartOf="@id/staking_benefits"
+                    app:layout_constraintTop_toBottomOf="@id/staking_receiving_rewards_message" />
+
+                <TextView
+                    android:id="@+id/staking_leaving_the_pool_title"
+                    style="@style/Subtitle2"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="10dp"
+                    android:text="@string/crowdnode_staking_leaving_the_pool_title"
+                    app:layout_constraintBottom_toBottomOf="@+id/staking_leaving_the_pool"
+                    app:layout_constraintStart_toEndOf="@id/staking_leaving_the_pool"
+                    app:layout_constraintTop_toTopOf="@id/staking_leaving_the_pool" />
+
+                <TextView
+                    android:id="@+id/staking_leaving_the_pool_message"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginEnd="15dp"
+                    android:text="@string/crowdnode_staking_leave_pool_message"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="@id/staking_leaving_the_pool_title"
+                    app:layout_constraintTop_toBottomOf="@id/staking_leaving_the_pool_title" />
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+            <TextView
+                android:id="@+id/staking_apy_title"
+                style="@style/Headline6"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="20dp"
+                android:layout_marginTop="30dp"
+                android:text="@string/crowdnode_staking_apy_title"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/staking_benefits_container" />
+
+            <TextView
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="20dp"
+                android:layout_marginTop="10dp"
+                android:text="@string/crowdnode_staking_current_apy_message"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/staking_apy_title" />
```

Comment:
This text seems to have the interline space too small. `Body2.Secondary` style should be used here to get the correct `lineSpacingExtra`.
In general, every `TextView` should have a style from the typography design system.

Reply:
Thanks.